projects/kit/components/textarea/textarea.template.html

```diff
@@ -0,0 +1,12 @@
+<ng-template #template>
```

Comment:
Why not `*tuiTextfieldContent` ?

Reply:
That way it is placed near icons, we want it placed right after the textarea in a different view container ref